PARAMETERS_MAPPING/parameters.csv

```diff
@@ -211,9 +211,9 @@ unique_id;cf_standard_name;cf_alias_name;long_name;imos_vocabulary_name;imos_voc
 861;;;Shannon diversity index;;;
 862;;;name of sampling station;;;
 863;;;ratio of Diatoms to Dinoflagellates;;;1 = all diatoms, 0 = all dinoflagellates
-864;mass_concentration_of_chlorophyll_in_sea_water;;Chlorophyll climatology for segment from MODIS 8 day 4 km monthly;Concentration of chlorophyll per unit volume of the water body;87;
-865;mass_concentration_of_chlorophyll_in_sea_water;;the best available Chlorophyll for segment from MODIS 4 km, 8 day, MODIS 4 km, monthly;Concentration of chlorophyll per unit volume of the water body;87;
-866;sea_water_temperature;;sea surface temperature for segment from GHRSST for post 2009 data, Reynolds weekly average from NOAA for pre 2009 data;Temperature of the water body;98;
+864;mass_concentration_of_chlorophyll_in_sea_water;;Chlorophyll climatology for segment from MODIS 4 km (8-day and monthly);Concentration of chlorophyll per unit volume of the water body;87;
+865;mass_concentration_of_chlorophyll_in_sea_water;;the best available Chlorophyll for segment from MODIS 4 km (8-day and monthly);Concentration of chlorophyll per unit volume of the water body;87;
```

Comment:
When we next show this header to Claire, we should specifically ask that she's happy with this.
Not knowing much about satellite products, it's not clear to me whether this changes the meaning or makes it more/less clear.

Reply:
yes ok, I just changed it in a way I could remove the commas because they were messing up the columns. But we can find another solution if they change the meaning